src/media-segment-request.js

```diff
@@ -367,6 +367,7 @@ const handleSegmentBytes = ({
     const {tracks} = segment.map;
 
     const trackInfo = {
+      isFmp4: true,
```

Comment:
This is how we determine if we are working with fmp4 or ts all the way up in master playlist loader.